include/z64transition.h

```diff
@@ -4,24 +4,31 @@
 #include "ultra64.h"
 #include "color.h"
 
+typedef enum {
+    /* 0 */ TRANS_TILE_OFF, // Inactive, do nothing
+    /* 1 */ TRANS_TILE_SETUP, // Save the necessary buffers
+    /* 2 */ TRANS_TILE_PROCESS, // Initalize the transition using the saved buffers
```

Comment:
```suggestion
    /* 2 */ TRANS_TILE_PROCESS, // Initialize the transition
```
the framebuffer isn't used at this point, it's only referenced in the built dlist, and that dlist is executed under `TRANS_TILE_READY`